hmpps-prisoner-search-indexer/helm_deploy/hmpps-prisoner-search-indexer/templates/remove-old-differences-cronjob.yaml

```diff
@@ -23,7 +23,7 @@ spec:
               args:
                 - /bin/sh
                 - -c
-                - curl --retry 2 -XDELETE http://hmpps-prisoner-search-indexer/prisoner-differences/delete
+                - curl --max-time 600 --verbose -XDELETE http://hmpps-prisoner-search-indexer/prisoner-differences/delete
```

Comment:
No need to retry: will try again tomorrow. Also increase timeout and set verbose for the time being to check behaviour in prod